templates/console/source/lambda/monitoring/realtime/add_partition/add_partition.py

```diff
@@ -7,14 +7,14 @@
 
 log = logging.getLogger()
 log.setLevel('INFO')
-client = boto3.client('glue')
-DB_NAME = os.environ['GLUE_DATABASE_NAME']
-TABLE_NAME = os.environ['GLUE_TABLE_NAME']
-CATALOG_ID = os.environ['ACCOUNT_ID']
-S3_URL = "s3://" + os.environ['S3_BUCKET']
 
 def lambda_handler(event, context):
 
+    client = boto3.client('glue')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

A Boto3 client is initialized from a Lambda handler. To speed up your Boto3 client initialization and minimize the operational cost of your Lambda function, we recommend that you create the client at the level of the module that contains the handler, and then reuse it between invocations.

[Learn more](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code)